stories/core/DatePicker.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { action } from '@storybook/addon-actions'
+import { DatePicker } from 'react-ui-core/src'
+
+export const DefaultDatePicker = (
+  <DatePicker
+    value="10/10/2018"
+    onClick={action('picked a date')}
+    placeholderText="Date"
+  />
+)
+
+export const DatePickerReadOnly = (
+  <DatePicker
+    value="11/01/2018"
+    onClick={action('picked a date')}
+    placeholderText="Date"
```

Comment:
nitpick: I think you meant `placeholder` - react-dom is complaining about this after getting spread on the input field.

Reply:
You're right! Not sure how that got in there 🤦‍♂️